index.js

```diff
@@ -2964,10 +2968,10 @@ module.exports = {
                 },
                 'q-applicant-other-treatment-dmi': {
                     type: 'string',
-                    title: 'Other treatments',
+                    title: 'Other mental health treatment',
```

Comment:
Line `3007` has not been changed in-line with all other error messages so far. Should line `3007` be changed from: 
`'Enter any other treatment you have received for your mental injury'`
to:
`'Enter any other mental health treatment you have received for your mental injury'`?

Reply:
Again, this was treated separately from the physical injuries and this content is in line with the data dictionary.